participating.md

```diff
@@ -0,0 +1,144 @@
+discuss it later during In The Weeds.
+
+## In The Weeds
+
+In The Weeds is a section for any longer discussion. Often it's to solicit
+community ideas or to find consensus on how to approach an issue. 
+
+> Issue with native busdevice
+>
+> * See hxxps://github.com/adafruit/circuitpython/issues/4109
+>   * Initially filed against LIS3DH library
+> * The issue is resolved if native BusDevice is not used.
+> * Can also resolve by building with LONGINT_IMPL = MPZ
+>   * But then it won’t fit in Flash unless something else is removed.
+
+
+
+# Before the meeting
+
+Remember to [check the meeting calendar](https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fraw.githubusercontent.com%2Fadafruit%2Fadafruit-circuitpython-weekly-meeting%2Fmaster%2Fmeeting.ical&title=CicuitPython%20Weekly%20Meeting%20Schedule&tab=agenda&tabs=month&tabs=agenda).
+The meeting is usually every Monday at 1PM [US Eastern Time (UTC-0400 or UTC-0500)](https://en.wikipedia.org/wiki/Eastern_Time_Zone),
+but we sometimes change the schedule for US holidays and other special events.
+If you're outside the US remember that the dates for the start and end of your
+"summer" time may differ from the rule followed in the US.
+
+Each meeting is accompanied by a notes document in Google Docs. You can find
+the link to the meeting document in Discord by looking in the "pinned
+messages". Generally the link is made available soon after the previous
+meeting is concluded.
+
+If you plan to give Hug Reports, Status Updates, or have an issue to discuss In
+The Weeds, you can update the notes doc at any time. For Hug Reports and
+Status Updates, please put your notes in alphabetical order, starting with your
+Discord handle.
+
+If the host should read your notes, indicate that next to your name, such as:
+
+> @foamyguy (lurking)
+
+> @tannewt (missing meeting)
+
+If you're going to be speaking and don't have time before the meeting to enter
+your notes, at least add your Discord handle as a placeholder so that the
+host knows to let you have your turn. If for some reason even that's not
+possible, ask in the text chat for someone else to add it for you.
+
+Your notes can be terse (they don't have to be full english sentences and
+paragraphs) but if the host is going to be reading them make sure they
+have enough context that what they say will make sense.
+
+# During the meeting
+
+Join a few minutes before the meeting if you want to check that your audio is working.
+
+Please give the host a few minutes after the start of the hour to arrive.
+The Discord meeting starts immediately after an internal Adafruit meeting.
+
+When not actively speaking, please mute your microphone.
+
+In the sections where you will participate, be attentive to the order of the
+meeting so that you're ready to un-mute and speak when called on.
+
+If the host accidentally skips you, place a note in the text chat to make
+them aware of it.
```

Comment:
Consider rewording this a bit. It might be a little unclear as is. My wording might not be the best either, but it's an idea.

```suggestion
If the host accidentally skips you,  please send a note in the Discord text chat to make
them aware of it.
```